apps/official/components/CovMapFeatureInfo.tsx

```diff
@@ -243,11 +231,7 @@ export const CovMapFeatureInfo = ({ rawData }: FeatureInfoProps) => {
   const cardContent = (
     <CardContent>
       <Grid container direction="column" spacing={2}>
-        {/* TODO: Comment this back in once the risk descriptions are updated */}
-        {/*<Grid item>*/}
-        {/*  <Typography>{riskDescription}</Typography>*/}
-        {/*</Grid>*/}
-        <Grid item xs={12}>
+        <Grid item xs={12} onClick={() => history.push(link)}>
```

Comment:
just for my understanding: Why are you using `history.push` instead of a `<Link to=...` here?

Reply:
My reasoning was: a Link also introduces styles, like underlined text, which have to be undone, while the history.push only does what it says. But since the other solution was already implemented. I changed it and use the Link inside the component now.